code/core/src/cli/eslintPlugin.test.ts

```diff
@@ -33,33 +33,33 @@ describe('extractEslintInfo', () => {
   } satisfies Partial<JsPackageManager>;
 
   beforeEach(() => {
-    vi.mocked(findUp).mockClear();
+    vi.mocked(find).up.mockClear();
```

Comment:
shouldn't this be `vi.mocked(find.up).mockClear()`?

presumably this works because `vi.mocked(find)` returns the mocked module, which has `up` which we set earlier as a `vi.fn`

Reply:
I don't think either makes a difference as long as `vi.mock("empathic/find")` has been called, as from my understanding it deeply creates a mock based on the module

Reply:
no worries that makes sense 👍